cdap-client/pom.xml

```diff
@@ -112,6 +112,10 @@
       <artifactId>cdap-explore-client</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
I'm not sure. Previously, it didn't know about Twill's RunId, right?
Is there a way to make it so that it doesn't know about the TwillRunId still, but only CDAP's RunId?